src/pathways/pathway/pathways.service.ts

```
@@ -28,6 +28,12 @@ import { CacheService } from 'src/cache/cache.service';
 import { CourseCompletionWebhookDto } from './dto/course-completion-webhook.dto';
 import { NotificationRequest } from '../../common/utils/notification.axios';
 
+const PATHWAY_SUBTYPE_PROGRAM_NAMES: Record<string, string> = {
+  CAL: 'Campus Leader Training!',
```

ESLint

[error] Replace `'Campus·Leader·Training!'` with `"Campus·Leader·Training!"`

(prettier/prettier)